Functions/Administration/xASL_adm_CorrectName.m

```diff
@@ -110,8 +110,11 @@
     end
     
 	% Report when the string was modified
-    if ~strcmp(strIn, strOut)
-		bCorrected = true;
+    if ~strcmp(strIn, strOut) 
+		% strcmp doesn't work properly when both strings have zero length, so we have to rule out this option before reporting a change
```

Comment:
Nicely catched. Next time I would add this as && instead of creating an extra condition, this is a smaller commit/change.

Reply:
I wanted to make it easier to understand. But fine with single line.

Done - you can resolve.